src/connector/src/common.rs

```diff
@@ -386,6 +386,47 @@ impl NatsCommon {
         Ok(subscription)
     }
 
+    pub(crate) async fn build_consumer(
+        &self,
+        start_sequence: Option<u64>,
+    ) -> anyhow::Result<
+        async_nats::jetstream::consumer::Consumer<async_nats::jetstream::consumer::pull::Config>,
+    > {
+        let context = self.build_context().await?;
+        let stream = self.build_or_get_stream(context.clone()).await?;
+        let name = "risingwave-consumer";
```

Comment:
Will it be a problem if consumers of different streams share the same name? My gut feeling is that it is better to use "risingwave-consumer-<stream_name>-<split_id>" as the consumer name.

Reply:
+1

Reply:
Solved. Change it in to `risingwave-consumer-<subject_name>-<split_id>`